templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
@ferventcoder what do you mean by "the mimicking platforms use"? This can be what makes the most sense for building nuget packages, as long as it's not too difficult to do programmatically :)